s3/compress.go

```diff
@@ -233,13 +234,21 @@ func (c *Compressor) Upload(ctx context.Context, body io.Reader, bucket, key *st
 func (c *Compressor) CreateObject(ctx context.Context, db models.DBInterface, body io.Reader, urlparams *models.URLParams, payload *models.ExportPayload) error {
 	filename := fmt.Sprintf("%s/%s/%s.%s", payload.OrganizationID, payload.ID, urlparams.ResourceUUID, payload.Format)
 
+	var failedUploads = prometheus.NewCounter(prometheus.CounterOpts{
+		Name: "failed_uploads_total",
+		Help: "The total number of failed S3 uploads.",
+	})
+
+	prometheus.MustRegister(failedUploads)
+
```

Comment:
Let's move this into a file named `s3/metrics.go`.

I think this might fail as its going to try to register the metric multiple times (each time `CreateObject()` is called).  We'll need to move the `MustRegister()` call into an init() method in the `s3/metrics.go` file.

We'll have to make `failUploads` a global (I don't like globals  :smile: ) but that's just the way it is.